tests/gtest/avif16bittest.cc

```diff
@@ -115,6 +119,16 @@ INSTANTIATE_TEST_SUITE_P(
         /*quality=*/
         testing::Values(AVIF_QUALITY_LOSSLESS)));
 
+INSTANTIATE_TEST_SUITE_P(
+    ResidualBitDepthExtension, SampleTransformTest,
```

Comment:
Nit: Add "s" to "ResidualBitDepthExtension" to make it plural? See "BitDepthExtensions" at line 113.